### PR TITLE
Add Cosmic Neutronium Block Solidifier Recipe

### PR DESCRIPTION
**Pull request by lordIcocain**
image

### CODE REVIEW

**Comment by EnderProyects** [comment]
I think black hole compressor using the fluid suits better to neutronium, maybe using a stack recipe? That make 64 or more at the time?

**Comment by lordIcocain** [comment]
why will use that if we have 
image

**Comment by EnderProyects** [comment]
Because is different fluid than ingots, also does the EOH give fluid neutronium? ( I don't remember that's why I think they are different in some way)

**Comment by FourIsTheNumber** [comment]
Fluid -> Block is really not suited to black hole, no. It doesn't do that kind of recipe and this would be wildly off-tier

**Comment by lordIcocain** [comment]
> Because is different fluid than ingots, also does the EOH give fluid neutronium? ( I don't remember that's why I think they are different in some way)

but you can mold ingot then compressor. EOH didnt give, you get neutronium`s from DTPF

File: src/main/java/com/dreammaster/gthandler/recipes/FluidSolidifierRecipes.java [hunk 1]
```diff
@@ -27,8 +28,12 @@
 
 public class FluidSolidifierRecipes implements Runnable {
 
+    ItemStack missing = new ItemStack(Blocks.fire);
```
Comment (chochem) [inline src/main/java/com/dreammaster/gthandler/recipes/FluidSolidifierRecipes.java:31]: why?

Comment (lordIcocain) [reply src/main/java/com/dreammaster/gthandler/recipes/FluidSolidifierRecipes.java:31]: As example i used CompressorRecipes.java. I repeated how it was done there.

Comment (chochem) [reply src/main/java/com/dreammaster/gthandler/recipes/FluidSolidifierRecipes.java:31]: 1) you are not using it 2) never use this for gt machine recipes, it just hides broken stuff

Comment (lordIcocain) [reply src/main/java/com/dreammaster/gthandler/recipes/FluidSolidifierRecipes.java:31]: fair.

**Review by FourIsTheNumber — APPROVED** [review]

**Pull request merged**